content/geoip/docs/databases/isp.md

```diff
@@ -102,63 +98,60 @@ respectively.
         <td>
           The organization associated with the registered autonomous system
           number for the IP address.
-
-          \* <a href="https://support.maxmind.com/hc/en-us/articles/4408200231067-IP-Network-Data#h_01FN989WWSE56YD6AV4QSPSWDW" target="_blank" rel="nofollow noopener noreferrer">Learn more about autonomous system data on our Knowledge Base.</a>
+          <br />
+          <a href="https://support.maxmind.com/hc/en-us/articles/4408200231067-IP-Network-Data#h_01FN989WWSE56YD6AV4QSPSWDW">Learn more about autonomous system data on our Knowledge Base.</a>
         </td>
     </tr>
     <tr>
       <td>mobile_country_code</td>
       <td>string</td>
       <td>
-        The [mobile country code
-        (MCC)](https://en.wikipedia.org/wiki/Mobile_country_code) associated
+        The <a href="https://en.wikipedia.org/wiki/Mobile_country_code">mobile country code
+        (MCC)</a> associated
         with the IP address and ISP.
-
-          \* <a href="https://support.maxmind.com/hc/en-us/articles/4408200231067-IP-Network-Data#h_01FT6Y6ANRH9GWYXE78B4RXAEX" target="_blank" rel="nofollow noopener noreferrer">Learn more about mobile country codes on our Knowledge Base.</a>
+        <br />
+        <a href="https://support.maxmind.com/hc/en-us/articles/4408200231067-IP-Network-Data#h_01FT6Y6ANRH9GWYXE78B4RXAEX">Learn more about mobile country codes on our Knowledge Base.</a>
       </td>
     </tr>
     <tr>
       <td>mobile_network_code</td>
       <td>string</td>
       <td>
-        The [mobile network code
-        (MNC)](https://en.wikipedia.org/wiki/Mobile_country_code) associated
+        The <a href="https://en.wikipedia.org/wiki/Mobile_country_code">mobile network code
+        (MNC)</a> associated
         with the IP address and ISP.
 
-          \* <a href="https://support.maxmind.com/hc/en-us/articles/4408200231067-IP-Network-Data#h_01FT6Y6ANRH9GWYXE78B4RXAEX" target="_blank" rel="nofollow noopener noreferrer">Learn more about mobile network codes on our Knowledge Base.</a>
+        <br />
+        <a href="https://support.maxmind.com/hc/en-us/articles/4408200231067-IP-Network-Data#h_01FT6Y6ANRH9GWYXE78B4RXAEX">Learn more about mobile network codes on our Knowledge Base.</a>
       </td>
     </tr>
 
   </tbody>
 </table>
+</div>
+{{</ rawhtml >}}
 
 ### Conversion Utility
 
-<ConversionUtility />
+{{< snippet "snippets/conversion-utility.md" >}}
 
 ## Example Files
 
-<ExampleFiles />
+{{< snippet "snippets/example-files.md" >}}
 
 ### CSV Example Files
 
-<CsvFileExamples files={[
-    {
-       filename: 'GeoIP2-ISP-CSV_Example.zip',
-       link: '/static/GeoIP2-ISP-CSV_Example.zip'
-    }
-  ]}
-/>
+We maintain examples of the CSV files as they would be downloaded from the account portal:
+
+* [GeoIP2-ISP-CSV\_Example.zip](/static/GeoIP2-ISP-CSV_Example.zip)
 
 ### MMDB Example Files
 
-<MmdbFileExamples files={[
```

Comment:
Can this be a snippet or is it just easier to not have one? Either way I think we can delete the `MmdbFileExamples` component once we delete the respective `anonymous-ip.mdx` and `enterprise.mdx` files.

Reply:
Hmm....  I suppose this could be a shortcode.  Something along the lines of
```
{{< mmdb-file-examples >}}
  * [some-file-1.zip](GeoIP some file 1)
  * [some-file-2.zip](GeoIP some file 2)
{{</ mmdb-file-examples >}}
```